src/middleware/validator/generic.validator.ts

```diff
@@ -6,7 +6,7 @@ import { GenericResponse } from '../../utils/apiResponse.utils'
 export function validate(req: Request, res: Response, next: NextFunction) {
   const errors = validationResult(req)
 
-  if (!errors.isEmpty()) return res.status(400).json({ errors: errors.array() })
+  if (!errors.isEmpty()) return res.status(400).json(errors.array())
```

Comment:
Why are we removing the `errors:` key from the body? Don't we want the response to have at least a semi-defined structure? i.e. (Ref example from [library example docs](https://express-validator.github.io/docs/))

```json
{
  "errors": [
    {
      "location": "body",
      "msg": "Invalid value",
      "param": "username"
    }
  ]
}
```